ctapipe/reco/hillas_intersection.py

```diff
@@ -43,6 +43,29 @@
     prefix="HillasIntersection",
 )
 
+FOV_ANGULAR_DISTANCE_LIMIT_RAD = (45 * u.deg).to_value(u.rad)
+
+
+def _far_outside_fov(fov_lat, fov_lon):
+    """
+    Check if a given latitude or longiude in the FoV is further away from
+    the FoV center than `FOV_ANGULAR_DISTANCE_LIMIT`
+
+    Parameters
+    ----------
+    fov_lat : u.Quantity[angle]
+        Latitude in TelescopeFrame or NominalFrame
+    fov_lon : u.Quantity[angle]
+        Longitude in TelescopeFrame or NominalFrame
+
+    Returns
+    -------
+    bool
+    """
+    lat_outside_fov = np.abs(fov_lat) > FOV_ANGULAR_DISTANCE_LIMIT_RAD
+    lon_outside_fov = np.abs(fov_lon) > FOV_ANGULAR_DISTANCE_LIMIT_RAD
+    return lat_outside_fov or lon_outside_fov
```

Comment:
fov_lat and fov_lon are spherical coordinates, so when talking about large angles, I think this should be defined as an angular distance on a sphere, not just a cut on lat and lon. Remember that in spherical coordinates, degrees in lat are in general smaller on the sky than degrees in lon, except a the equator, so just comparing them each like this will give different results for different latitudes. 

It might  be better to use `SkyCoord.separation()` instead

Reply:
```py3
point = SkyCoord(fov_lat, fov_lon, frame=TelescopeFrame)
center = SkyCoord(0*u.deg, 0*u.deg, frame=TelescopeFrame)
return point.angular_separation(center).to_value(rad) > FOV_ANGULAR_DISTANCE_LIMIT_RAD
```

Reply:
>  Remember that in spherical coordinates, degrees in lon are in general smaller on the sky than degrees in lat, except a the equator (here the center of the FOV),

We are at the equator (or even more precisely the crossing of zero meridian and equator) and want to remove things that are far away from it, so for all intents and purposes, I think we can treat lat and lon as equivalent here.

It's also not something we need to precise about, since it is really just some arbitrary limit to prevent an error from happening. 

computing the full angular separation will be much more expensive.

If you think it is really necessary, it would be much faster to call it directly using `astropy.coordiantes.angle_utilities.angular_separation` instead of constructing skycoordinates.

However, I don't think we need to care about cutting out a circle vs. a square here and should chose the fastest, sensible method.

Reply:
````
In [10]: np.rad2deg(angular_separation(np.pi /4, 0, 0, 0))
Out[10]: 45.0

In [11]: np.rad2deg(angular_separation(np.pi /2, 0, 0, 0))
Out[11]: 90.0

In [12]: np.rad2deg(angular_separation(0, np.pi/2, 0, 0))
Out[12]: 90.0

In [13]: np.rad2deg(angular_separation(0, np.pi/4, 0, 0))
Out[13]: 45.0
```

Reply:
@kosack Could you comment if you think it is necessary for this check to use true angular separation or the current, slightly faster solution is fine?